optifit backend/pushup_counter.py

```
@@ -0,0 +1,353 @@
+    # Encode to H.264 using ffmpeg
+    print("🎞 Converting push-up video to H.264 using ffmpeg...")
+    ffmpeg_cmd = [
+        'ffmpeg', '-y',
+        '-i', raw_path,
+        '-vcodec', 'libx264',
+        '-preset', 'fast',
+        '-crf', '23',
+        '-acodec', 'aac',
+        output_path
+    ]
+    subprocess.run(ffmpeg_cmd, check=True)
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)